docs/user-guide/admin-cli.md

```diff
@@ -87,45 +87,106 @@ optional arguments:
+vmgroup 'vmgroup1' is created. Do not forget to run 'vmgroup vm add' to add vm to vmgroup.
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
-Uuid                                  Name       Description                 Default_datastore  VM_list
-------------------------------------  ---------  --------------------------  -----------------  -------
-11111111-1111-1111-1111-111111111111  _DEFAULT   This is a default vmgroup
-1ddb5b46-6a9f-4649-8e48-c47039905752  vmgroup1
+Uuid                                  Name      Description                Default_datastore  VM_list
+------------------------------------  --------  -------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup  __VM_DS
+9de84179-6894-44ad-b444-470e8619a5ed  vmgroup1                             datastore1
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
+Datastore   Allow_create  Max_volume_size  Total_size
+----------  ------------  ---------------  ----------
+datastore1  True          Unset            Unset
+```
+
+The "default_datastore" can be also set to a special value "__VM_DS" during vmgroup create.
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                  Default_datastore  VM_list
+------------------------------------  --------  ---------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is the default vmgroup  __VM_DS
+
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup create --name=vmgroup1 --default-datastore="__VM_DS"
+vmgroup 'vmgroup1' is created. Do not forget to run 'vmgroup vm add' to add vm to vmgroup.
+[root@localhost:~]
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                  Default_datastore  VM_list
+------------------------------------  --------  ---------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is the default vmgroup  __VM_DS
+30545fdc-20e0-409a-8330-6ebe027fcc34  vmgroup1                               __VM_DS
+
+[root@localhost:~]
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
+Datastore  Allow_create  Max_volume_size  Total_size
+---------  ------------  ---------------  ----------
+__VM_DS    True          Unset            Unset
+
+```
+
+The "default_datastore" cannot be set to "__ALL_DS" and an error will be generated when trying to set during vmgroup create.
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup create --name=vmgroup2 --default-datastore="__ALL_DS"
+Default datastore name __ALL_DS is invalid
```

Comment:
The message need to be a bit better "Cannot use __ALL_DS as default datastore. Please use specific datastore name or _VM_DS special datastore"